static/app/views/performance/transactionSummary/utils.tsx

```diff
@@ -20,8 +20,14 @@ export enum TransactionFilterOptions {
   RECENT = 'recent',
 }
 
-export function generateTransactionSummaryRoute({orgSlug}: {orgSlug: string}): string {
-  return `/organizations/${orgSlug}/performance/summary/`;
+export function generateTransactionSummaryRoute({
+  orgSlug,
+  subPath,
+}: {
+  orgSlug: string;
+  subPath?: string;
+}): string {
+  return `/organizations/${orgSlug}/performance/summary/${subPath ? subPath + '/' : ''}`;
```

Comment:
Nit, but I do think it looks cleaner even with the nested template strings
```suggestion
  return `/organizations/${orgSlug}/performance/summary/${subPath ? `${subPath}/` : ''}`;
```

Reply:
for sure! thanks!